dev-tools/notice/overrides.json

```diff
@@ -11,3 +11,4 @@
 {"name": "github.com/munnerz/goautoneg", "licenceType": "BSD-3-Clause"}
 {"name": "github.com/pelletier/go-buffruneio", "licenceType": "MIT"}
 {"name": "github.com/urso/magetools", "licenceType": "Apache-2.0"}
+{"name": "kernel.org/pub/linux/libs/security/libcap/cap", "licenseFile": "License", "licenseType": "BSD-3-Clause", "note": "Dual licensed as BSD-3-Clause / GPL-2.0. See https://git.kernel.org/pub/scm/libs/libcap/libcap.git/tree/License"}
```

Comment:
Do we have any license issue here? (not an expert, sorry)

Reply:
No, it's dual licensed, you can pick which license you want, so we pick BSD-3-Clause of course. For some reason the licensing tool isn't detecting this override though.

Reply:
I'm not familiar with https://github.com/elastic/go-licence-detector, but based on how the overrides are described in the README I wonder if it only checks the overrides in case of failure and perhaps it's not "failing" and just thinks it's GPL2.

Reply:
I did fix this, the cause was you can't specify `licenseType` and `licenseFile`.